packages/loot-core/src/platform/client/fetch/index.web.ts

```diff
@@ -78,11 +78,7 @@ export const init: T.Init = async function () {
   return new Promise(connectSocket);
 };
 
-export const send: T.Send = function (
-  name,
-  args,
-  { catchErrors = false } = {},
-) {
+export const send = function (name, args, { catchErrors = false } = {}) {
   return new Promise((resolve, reject) => {
```

Comment:
Can you try below to see if this can fix the return value type? I believe the same can be done on the index.browser.ts
```suggestion
export const send: T.Send = function (
  ...params: Parameters<T.Send>
): ReturnType<T.Send> {
  const [name, args, { catchErrors }] = params;
  return new Promise((resolve, reject) => {
```

Reply:
Yeah, this works!